### DISCUSSION TITLE
Assign multiple roles to a user

### DISCUSSION THREAD

**Discussion by slzno** [spatie/laravel-permission#1960] [Ideas]
I want to assign multiple roles to a user, but doing so this conflicts with permissions.

**Comment by erikn69** [comment]
>but doing so this conflicts with permissions.

Why? Environment? Versions? steps to reproduce? Examples? Nothing?
 
------
That is tested
https://github.com/spatie/laravel-permission/blob/79f7dbf2b6b21d81209774f5726a0e9f4222dd92/tests/HasRolesTest.php#L96-L103
